java/clients/src/test/java/sleeper/status/report/filestatus/FileInfoFactory.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.status.report.filestatus;
+
+import sleeper.core.key.Key;
+import sleeper.core.partition.Partition;
+import sleeper.core.partition.PartitionTree;
+import sleeper.core.schema.Schema;
+import sleeper.statestore.FileInfo;
+
+import java.time.Instant;
+import java.util.List;
+
+public class FileInfoFactory {
+    private final Schema schema;
+    private final PartitionTree partitionTree;
+    private final Instant lastStateStoreUpdate;
+
+    public FileInfoFactory(Schema schema, List<Partition> partitions, Instant lastStateStoreUpdate) {
+        this.schema = schema;
+        this.lastStateStoreUpdate = lastStateStoreUpdate;
+        partitionTree = new PartitionTree(schema, partitions);
```

Comment:
I would make this "this.partitionTree" for consistency with the lines above.

Reply:
Done